### ISSUE TITLE
Possible to rerun cloning without destroying secrets/service

### ISSUE THREAD

**Issue by GoldenMouse** [zalando/postgres-operator#1462]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?** egistry.opensource.zalan.do/acid/postgres-operator:v1.6.2
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?**  AWS EKS
- **Are you running Postgres Operator in production?** no
- **Type of issue?**  question

Is it possible to rerun the database cloning without destroying the pg resource or reinstalling?  I figure out that one can run the `python3 /scripts/clone_with_basebackup.py` but after this the postgres admin password no longer works.  

We need to reinitialize the db from time to time. I'm trying to do this while avoiding recreating the secret and load balancer because this breaks connection with external apps the pg is currently talking to.

**Comment by FxKu** [comment]
No, once a clone is initialized and Postgres is running you cannot rerun the cloning on the existing cluster. Can you not create a new cluster each time and then redeploy your app switching traffic to the new cluster? Sound automatizable to me.
